### PR TITLE
add k8s deployment files

### PR DESCRIPTION
**Pull request by pranaysahith**

### CODE REVIEW

File: deployment/kubernetes/skaffold.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: skaffold/v2beta4
+kind: Config
+metadata:
+  name: fgto-eventuate-tram-examples-customers-and-orders
```
Comment (cer) [inline deployment/kubernetes/skaffold.yaml:4]: Remove `fgto-` prefix

File: deployment/kubernetes/yaml/cdcservice.yaml [hunk 2]
```diff
@@ -0,0 +1,133 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ftgo-cdc-service
```
Comment (cer) [inline deployment/kubernetes/yaml/cdcservice.yaml:4]: This application is called `eventuate-tram-examples-customers-and-orders` - not `ftgo` (name of the other application). However, that's a long name. I'd remove the `ftgo-` prefix for all k8s objects and define an `eventuate-tram-examples-customers-and-orders` namespace

File: deployment/kubernetes/yaml/cdcservice.yaml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      containers:
+      - name: ftgo-cdc-service
+        image: eventuateio/eventuate-cdc-service:0.4.0.RELEASE
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8080
+          name: httpport
+        command: ["bash", "-c", "java -Dsun.net.inetaddr.ttl=30 -jar *.jar" ]
+        env:
+          - name: SPRING_DATASOURCE_URL
```
Comment (cer) [inline deployment/kubernetes/yaml/cdcservice.yaml:41]: Please set up the `env:` based on the environment variables defined in the `eventuate-tram-examples-customers-and-orders/docker-compose-mysql-binlog.yml` file

File: deployment/kubernetes/yaml/customerservice.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    targetPort: 8080
+  selector:
+    svc: ftgo-customer-service
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ftgo-customer-service
+  labels:
+    application: ftgo
```
Comment (cer) [inline deployment/kubernetes/yaml/customerservice.yaml:18]: The application should be `eventuate-tram-examples-customers-and-orders` - not `ftgo`

File: deployment/kubernetes/yaml/mongodb.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  serviceName: ftgo-mongodb
+  template:
+    metadata:
+      labels:
+        app: ftgo-mongodb
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: ftgo-mongodb
+        image: mongo:4.4.0-rc8-bionic
```
Comment (cer) [inline deployment/kubernetes/yaml/mongodb.yaml:30]: Please use the mongo image in the `docker-compos....` file.

File: deployment/kubernetes/yaml/mongodb.yaml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+          - name: ftgo-mongo-persistent-storage
+            mountPath: "/data/db"
+  volumeClaimTemplates:
+  - metadata:
+      name: ftgo-mongo-persistent-storage
+    spec:
+      accessModes: ["ReadWriteOnce"]
+      resources:
+        requests:
+          storage: 1Gi
```
Comment (cer) [inline deployment/kubernetes/yaml/mongodb.yaml:49]: The volumes be much smaller - 256m?

File: deployment/kubernetes/yaml/mysql.yaml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        role: ftgo-mysql
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name: ftgo-mysql
+          image: msapatterns/mysql:latest
```
Comment (cer) [inline deployment/kubernetes/yaml/mysql.yaml:33]: Please use the image from `eventuate-tram-examples-customers-and-orders/docker-compose-mysql-binlog.yml`

File: deployment/kubernetes/yaml/orderhistoryservice.yaml [hunk 8]
```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ftgo-orderhistoryservice
```
Comment (cer) [inline deployment/kubernetes/yaml/orderhistoryservice.yaml:4]: Please use hyphens in the service names and YAML file names: `order-history-service`

File: deployment/kubernetes/yaml/orderhistoryservice.yaml [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+            value: ftgo-zookeeper:8121
+          - name: SPRING_SLEUTH_ENABLED
+            value: "true"
+          - name: SPRING_SLEUTH_SAMPLER_PROBABILITY
+            value: "1"
+          - name: SPRING_ZIPKIN_BASE_URL
+            value: http://zepkin:9411
+        livenessProbe:
+          httpGet:
+            path: /actuator
```
Comment (cer) [inline deployment/kubernetes/yaml/orderhistoryservice.yaml:50]: The path for all Java services should be `/actuator/health`

File: deployment/kubernetes/yaml/orderhistoryservice.yaml [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+          - name: EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS
+            value: ftgo-kafka:9092
+          - name: EVENTUATELOCAL_ZOOKEEPER_CONNECTION_STRING
+            value: ftgo-zookeeper:8121
+          - name: SPRING_SLEUTH_ENABLED
+            value: "true"
+          - name: SPRING_SLEUTH_SAMPLER_PROBABILITY
+            value: "1"
+          - name: SPRING_ZIPKIN_BASE_URL
+            value: http://zepkin:9411
```
Comment (cer) [inline deployment/kubernetes/yaml/orderhistoryservice.yaml:47]: zipkin is mispelt

File: deployment/kubernetes/skaffold.yaml (hunk 1 above)
Comment (pranaysahith) [reply deployment/kubernetes/skaffold.yaml]: removed fgto- prefixes everywhere

File: deployment/kubernetes/yaml/cdcservice.yaml (hunk 2 above)
Comment (pranaysahith) [reply deployment/kubernetes/yaml/cdcservice.yaml]: removed fgto- prefixes and deployed to `eventuate-tram-examples-customers-and-orders`namespace

File: deployment/kubernetes/yaml/cdcservice.yaml (hunk 3 above)
Comment (pranaysahith) [reply deployment/kubernetes/yaml/cdcservice.yaml]: updated

File: deployment/kubernetes/yaml/customerservice.yaml (hunk 4 above)
Comment (pranaysahith) [reply deployment/kubernetes/yaml/customerservice.yaml]: updated

File: deployment/kubernetes/yaml/mongodb.yaml (hunk 5 above)
Comment (pranaysahith) [reply deployment/kubernetes/yaml/mongodb.yaml]: updated

File: deployment/kubernetes/yaml/mysql.yaml (hunk 7 above)
Comment (pranaysahith) [reply deployment/kubernetes/yaml/mysql.yaml]: updated

File: deployment/kubernetes/yaml/orderhistoryservice.yaml (hunk 8 above)
Comment (pranaysahith) [reply deployment/kubernetes/yaml/orderhistoryservice.yaml]: updated service names and file names.

File: deployment/kubernetes/yaml/orderhistoryservice.yaml (hunk 9 above)
Comment (pranaysahith) [reply deployment/kubernetes/yaml/orderhistoryservice.yaml]: updated

File: deployment/kubernetes/yaml/orderhistoryservice.yaml (hunk 10 above)
Comment (pranaysahith) [reply deployment/kubernetes/yaml/orderhistoryservice.yaml]: updated

File: deployment/kubernetes/yaml/mongodb.yaml (hunk 6 above)
Comment (pranaysahith) [reply deployment/kubernetes/yaml/mongodb.yaml]: updated

**Comment by cer** [comment]
Merged into https://github.com/eventuate-tram/eventuate-tram-examples-customers-and-orders/tree/wip-k8s

**Pull request closed without merging**